R/shuffle_ard.R

```diff
@@ -223,8 +224,119 @@ shuffle_ard <- function(x, trim = TRUE) {
   }
 }
 
+#' Rename ARD Columns
+#'
+#' This function combines a pair of `group`/`group_level` or `variable`/`variable_level` columns into a
+#' single column. The `group_level` or `variable_level` column is renamed according to the value of
+#' the `group` or `variable` column, respectively.
+#'
+#' @param x (`data.frame`)\cr
+#'   a data frame
+#' @param col ([`tidy-select`][dplyr::dplyr_tidy_select])\cr
+#'  Name of column containing the variable names
+#' @param col_lev ([`tidy-select`][dplyr::dplyr_tidy_select])\cr
+#'  Name of column containing the variable levels. If `NULL` (default), this will be assumed to match the `col` argument with `_level` appended to the end (e.g. if `col` = "group1", `col_lev` will be assumed to be "group1_level")
+#'
+#' @return data frame
+#' @export
+#'
+#' @examples
+#' data <- data.frame(group1 = "A", group1_level = "B", variable = "C", variable_level = "D")
+#'
+#' cards:::rename_ard_columns(data, "group1")
```

Comment:
Since this is exported, we don't need `:::` in the examples

Reply:
Ah yes, thanks for catching!